<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>625</width>
         <height>417</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabelEX" name="imgLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImage_Processing">
    <property name="title">
     <string>Image Processing</string>
    </property>
    <addaction name="actionOpen_BMP_file"/>
    <addaction name="actionSave_to_new_BMP_file"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_file_header"/>
    <addaction name="actionGet_pixel_value"/>
    <addaction name="actionSet_pixel_value"/>
    <addaction name="separator"/>
    <addaction name="actionImage_interpolation"/>
    <addaction name="actionMedian_filtering"/>
    <addaction name="actionGaussian_smoothing"/>
    <addaction name="actionImage_Rotate"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>视图(V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <addaction name="menuImage_Processing"/>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_BMP_file">
   <property name="text">
    <string>Open BMP file</string>
   </property>
  </action>
  <action name="actionSave_to_new_BMP_file">
   <property name="text">
    <string>Save to new BMP file</string>
   </property>
  </action>
  <action name="actionDisplay_file_header">
   <property name="text">
    <string>Display file header</string>
   </property>
  </action>
  <action name="actionGet_pixel_value">
   <property name="text">
    <string>Get pixel value</string>
   </property>
  </action>
  <action name="actionSet_pixel_value">
   <property name="text">
    <string>Set pixel value</string>
   </property>
  </action>
  <action name="actionImage_interpolation">
   <property name="text">
    <string>Image interpolation</string>
   </property>
  </action>
  <action name="actionMedian_filtering">
   <property name="text">
    <string>Median filtering</string>
   </property>
  </action>
  <action name="actionGaussian_smoothing">
   <property name="text">
    <string>Gaussian smoothing</string>
   </property>
  </action>
  <action name="actionHistogram_equalization">
   <property name="text">
    <string>Histogram equalization</string>
   </property>
  </action>
  <action name="actionSharpening_by_gradient">
   <property name="text">
    <string>Sharpening by gradient</string>
   </property>
  </action>
  <action name="actionBilateral_filtering">
   <property name="text">
    <string>Bilateral filtering</string>
   </property>
  </action>
  <action name="actionAdd_impulse_noise">
   <property name="text">
    <string>Add impulse noise</string>
   </property>
  </action>
  <action name="actionCanny_edge_detectio">
   <property name="text">
    <string>Canny edge detection</string>
   </property>
  </action>
  <action name="actionImage_Rotate">
   <property name="text">
    <string>image Rotate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLabelEX</class>
   <extends>QLabel</extends>
   <header location="global">qlabelex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
